src/test/java/com/trello/trello/stepdefs/BoardSteps.java

```diff
@@ -0,0 +1,44 @@
+package com.trello.trello.stepdefs;
+
+import com.trello.trello.entities.Context;
+import com.trello.trello.ui.pages.BoardPage;
+import core.selenium.WebDriverHelper;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.testng.Assert;
+
+public class BoardSteps {
+
+    private Context context;
+    private BoardPage board;
+
+    /**
+     * Constructor of a Board.
+     * @param cont
+     */
+    public BoardSteps(final Context cont) {
+        board = new BoardPage();
```

Comment:
You should not instantiate a page in the constructor, only when it is displayed in the application